orcestra/flightplan.py

```diff
@@ -389,3 +389,60 @@ def path_quickplot(path, sel_time, crossection=True):
             ax2.axvline(point, color="k", lw=1)
 
     return fig
+
+def calc_zonal_mean(field, lon_min, lon_max, lat_min, lat_max): 
+    import easygems.healpix as egh
+    
+    bbox = [lon_min, lon_max, lat_min, lat_max]
+    
+    if "lon"  in field.coords:
+        field = field.rename({'lon': 'longitude'})
+    if "lat"  in field.coords:
+        field = field.rename({'lat': 'latitude'})
+    
+    in_bbox = egh.isel_extent(field, bbox) 
+
+    field_bbox = field.sel(
+        cell=in_bbox,
+    )
+    
+    field_lat = field_bbox.groupby(
+        field_bbox.latitude
+    ).mean(
+    )
+    return field_lat
+
+def find_edges(cwv, cwv_thresh, cwv_min = 0, lat_cwv_max = 9.0):
+    
+    import scipy.signal
```

Comment:
This line adds a dependency on `scipy`. Can you add this to `pyproject.toml` as well, such that `scipy` will be installed automatically when the `orcestra` package is installed.